tests/validations/mta/mtaValidations.spec.ts

```diff
@@ -18,13 +18,22 @@ import { Diagnostic, Range, Uri } from "vscode";
 import * as mtaDiagnostic from "../../../src/validations/mta/mtaDiagnostic";
 import { resolve } from "path";
 import * as fsExtra from "fs-extra";
-import { mta, Mta } from "@sap/mta-lib";
+import Mta, { mta } from "@sap/mta-lib";
 import { forEach } from "lodash";
 
+interface stubPathExistsInfo {
+  path: string;
+  resolvesTo: boolean;
+}
+
 describe("mtaValidations", () => {
   const disposables: Disposable[] = [];
-  const mtaYamlPath = resolve(__dirname, MTA_YAML);
-  const devMtaExtPath = resolve(__dirname, DEV_MTA_EXT);
+  const modulePath = __dirname;
```

Comment:
It should be `projectPath`. The mta.yaml is a project (application) definition, we don't check the modules in it.